rai/client_test.go

```diff
@@ -95,7 +95,7 @@ func tearDown(client *Client) {
 
 	user, _ := client.FindUser(userEmail)
 	if user != nil {
-		defer client.DeleteUser(user.ID)
+		client.DeleteUser(user.ID)
```

Comment:
check error? maybe
```suggestion
		assert.Nil(client.DeleteUser(user.ID))
```

Reply:
I think we can't use `assert` at the level of this function :pray: